### PR TITLE
Use semantic colours in the TransactionView

### PR DESCRIPTION
**Pull request by robinsalehjan**
# Why?

- Did not look good with dark mode.

# What?

- Use semantic colors for the `TransactionStepView+Style`
- Always use `.licorice` for `TransactionWarningView`

# Show me

| Basic | Dark |
| --- | --- |
| Simulator Screen Shot - iPhone 11 Pro - 2020-03-04 at 09 54 37 | Simulator Screen Shot - iPhone 11 Pro - 2020-03-04 at 09 52 00 |

### CODE REVIEW

File: Sources/Fullscreen/TransactionView/TransactionStepView/TransactionStepView+Style.swift [hunk 1]
```diff
@@ -38,9 +38,9 @@ public extension TransactionStepView {
         var titleTextColor: UIColor {
             switch self {
             case .notStarted:
-                return .stone
+                return .textSecondary
             default:
```
Comment (bstien) [inline Sources/Fullscreen/TransactionView/TransactionStepView/TransactionStepView+Style.swift]: Shouldn't this one be exhaustive as well?

Comment (bstien) [reply Sources/Fullscreen/TransactionView/TransactionStepView/TransactionStepView+Style.swift]: I see the ones for `bodyTextColor` and `detailTextColor` list all cases

Comment (robinsalehjan) [reply Sources/Fullscreen/TransactionView/TransactionStepView/TransactionStepView+Style.swift]: The title stays the same regardless of state (atleast for now) 😄

Comment (robinsalehjan) [reply Sources/Fullscreen/TransactionView/TransactionStepView/TransactionStepView+Style.swift]: The only special case is if it's `notStarted`

**Pull request merged**